homeassistant/components/abode/__init__.py

```diff
@@ -259,6 +259,12 @@ def event_callback(event, event_json):
         TIMELINE.PANEL_FAULT_GROUP,
         TIMELINE.PANEL_RESTORE_GROUP,
         TIMELINE.AUTOMATION_GROUP,
+        TIMELINE.DISARM_GROUP,
```

Comment:
We shouldn't add events that can be derived from the state of the alarm control panel entity changing.

Reply:
The events provide more information than just the state change, notably the user who accomplished the action. I can see a few places where this would be useful, such as doing different things if children, landlords, contractors, cleaners, etc. disarm the system.